msteams-platform/bots/how-to/conversations/conversation-basics.md

```diff
@@ -25,7 +25,7 @@ A bot behaves differently depending on the conversation it's involved in:
 * Bots in a one-to-one conversation don't require an @mention. All messages sent by the user routes to your bot.
 
 > [!NOTE]
-> Bots can be enabled to receive all channel messages in a team without being @mentioned using resource-specific consent (RSC) permissions. This feature is currently available in [public developer preview](../../../resources/dev-preview/developer-preview-intro.md) only. For more information, see [receive all channel messages with RSC](channel-messages-with-rsc.md).
+> Bots can receive all channel and chat messages in a team without being @mentioned using resource-specific consent (RSC) permissions. RSC for all *chat* messages is available in [public developer preview for Teams](../../../resources/dev-preview/developer-preview-intro.md). For more information, see [receive all channel messages with RSC](channel-messages-with-rsc.md).
```

Comment:
Minor modification. Please update with below:

Bots can receive all channel and chat messages in a team without being @mentioned using resource-specific consent (RSC) permissions. RSC for all *chat* messages is currently only available in [public developer preview for Teams](../../../resources/dev-preview/developer-preview-intro.md). For more information, see [receive all channel messages with RSC](channel-messages-with-rsc.md).

Reply:
Updated as suggested.